besser/bot/core/state.py

```diff
@@ -141,14 +144,17 @@ def go_to(self, dest: 'State') -> None:
         """Create a new `auto` transition on this state.
 
         This transition needs no event to be triggered, which means that when the bot moves to a state that has an
-        `auto` transition, the bot will move to the transition's destination state unconditionally.
+        `auto` transition, the bot will move to the transition's destination state unconditionally without waitng for user input.
+        This transition cannot be combined with other transitions.
 
         Args:
             dest (State): the destination state
         """
         for transition in self.transitions:
             if transition.is_auto():
                 raise DuplicatedAutoTransitionError(self._bot, self)
+            if not transition.is_auto():
+                raise ConflictingAutoTransitionError(self._bot, self)
```

Comment:
I would change this code, since now we could simplify the problem and simply check if the state has already any transition. If that is the case, raise an exception since the go_to transition does not allow more transitions in a state